conversion/conversion.go

```diff
@@ -538,32 +538,56 @@ func getInstances(project string) ([]string, error) {
 
 // WriteSchemaFile writes DDL statements in a file. It includes CREATE TABLE
 // statements and ALTER TABLE statements to add foreign keys.
+// The parameter name should end with a .txt.
 func WriteSchemaFile(conv *internal.Conv, now time.Time, name string, out *os.File) {
 	f, err := os.Create(name)
 	if err != nil {
 		fmt.Fprintf(out, "Can't create schema file %s: %v\n", name, err)
 		return
 	}
-	// The schema file we write out includes comments, includes foreign keys
-	// and doesn't add backticks around table and column names. This file is
-	// intended for explanatory and documentation purposes, and is not strictly
-	// legal Cloud Spanner DDL (Cloud Spanner doesn't currently support comments).
```

Comment:
I think we should keep this part of the comment here with slightly modification. Something on the lines of:

// The schema file written below is optimized for reading. It includes comments, foreign keys and doesn't
// add backticks around table and column names. Since this file is intended for explanatory and documentation 
// purposes, it is not strictly legal Cloud Spanner DDL (Cloud Spanner doesn't currently support comments).